### DISCUSSION TITLE
Black circles are effectively invisible in dark mode

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44176] [Mobile] [answered]
GitHub Mobile beta

Trending applications > filter by language

![Screenshot_20230112-223856.png](https://user-images.githubusercontent.com/2119212/212231808-5f37896d-be22-4b9d-98f4-816eefb14ca2.png)

To use black circles in dark mode, a white circle should be included around them.

**Comment by stevepopovich** [comment, marked as answer]
Hi @jsoref, thanks for the feedback. I have filed this for the team to look at.
